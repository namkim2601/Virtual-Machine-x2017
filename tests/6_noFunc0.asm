FUNC LABEL 7
    PRINT VAL 8
    RET
