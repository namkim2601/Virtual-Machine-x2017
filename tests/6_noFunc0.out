Error: Entry point of program can not be found, please provide a function with the label 0
